docs/sanic/response.md

```diff
@@ -0,0 +1,99 @@
+# Response
+
+You can use functions in `sanic.response` module to create response.
```

Comment:
Sorry to be picky, but how about "Use functions in `sanic.response` module to create responses." instead?